File: Python/Lab_3/Lab_3.py
accuracy = float(input("Введіть точність обчисленння: "))
e = 0
i = 0
fact = 1
current = 1 / fact  
while current > accuracy:
        e = e + current
        i = i + 1
        fact = fact * i
        current = 1 / fact
    
print("e =", e)

File: С#/Lab_3/Lab_3.cs
using System;

namespace Lab_3
{
    class Lab_3
    {
        static void Main(string[] args)
        {
            Console.Write("Enter accuracy: ");
            double accuracy = Convert.ToDouble(Console.ReadLine());
            double e = 1;
            int i = 0;
            int fact = 1;
            double current = 0;
            do
            {
                e += current;
                i += 1;
                fact *= i;
                current = 1.0 / fact;
            } while (current > accuracy);

            Console.WriteLine($"e = {e}");
        }
    }
}
